main.py

```diff
@@ -0,0 +1,18 @@
+def palindrome():
+  word=input('Check to see if word is a palindrome: ')
+  #Grabbing the word from the user
+
+
+  reverse = ''.join(reversed(word))
+  #Joining the input withouth any seperators and reversing the list of letters and saving it as a variable
+
+
+  if(word == reverse):
+    #Checking if the two match
+    print('this is a palindrome')
+  else:
+    print('Sorry not a palindrome')
+
+
+
+palindrome()
```

Comment:
Great start on this Kofi! I like the spin you put on reversing strings. Really clean code it! How could we use this function if we gave you an array of palindromes and non-palindromes? Something to think about. Really great start though!